playbooks/ansible-test-base/pre.yaml

```
@@ -47,4 +47,4 @@
         name: deploy-artifacts
       when:
         - ansible_test_collections | default(False)
-        - "'integration' not in ansible_test_command"
+      #  - "'integration' not in ansible_test_command"
```

```suggestion
        #- "'integration' not in ansible_test_command"
```
I see a warning [comment not indented like content](https://dashboard.zuul.ansible.com/t/ansible/build/ad19dfd1118f4fae8d48299b347fe799), maybe this will fix it.